server/monitor.go

```diff
@@ -1910,3 +1913,161 @@ func (reason ClosedState) String() string {
 
 	return "Unknown State"
 }
+
+// LeafzOptions are options passed to Leafz
+type AccountzOptions struct {
+	// Account indicates that Accountz will return details for the account
+	Account string `json:"account"`
+}
+
+type ExtImport struct {
+	jwt.Import
+	Invalid bool
+}
+
+type ExtExport struct {
+	jwt.Export
+	ApprovedAccounts []string `json:"approved_accounts"`
```

Comment:
Here and almost all of them in AccountInfo, you seem to not use `,omitempty`. Is there a reason? The boolean and int, if 0, could be omitted, which will make the JSON marshal smaller. Same for list of exports and imports (Exps, Imps, which by the way could probably be spelled out: `Exports` and `Imports`.

Reply:
I used omitempty sparingly because I imagine this to be used for debugging.
For this use case I imagine having values be explicitly listed makes more sense. 
I used it only in cases where the source of the account makes a difference (jwt or config), which is also why I'll add it to this field.

Reply:
But I'll can add it everywhere. Your call.

Reply:
But if explicitly listed but equal to the type uninitialized value (false for boolean, 0 for numbers, etc), you would really not know the difference. Meaning if you see say "expired: false", I am not even sure what that would mean to be set explicitly.

Or here, `approved_accounts`, if you don't see it it means it was nil. What difference (except for marshal result being bigger) would that make to get `approved_accounts` received as `[]string{}` (or nil). 

If you could give me an example of a case where having without `omitempty` would help debug as opposed to if the field had `omitempty`.

Reply:
Basically cut down on time spent trying to find out what should be there. If you don't think that matters I'll put omitempty on every field.

approved_accounts I added it already.